### PR TITLE
reapply bfs deltas from 2.5.9

### PR DESCRIPTION
**Pull request by markxnelson**
Signed-off-by: Mark Nelson <mark.x.nelson@oracle.com>

### CODE REVIEW

File: Dockerfile.dapper [hunk 1]
```diff
@@ -10,28 +13,48 @@ ENV CATTLE_HELM_UNITTEST_VERSION v0.1.7-rancher3
...
+    git version
+
+# Update PATH to make sure git 2.27 is on the path
+ENV PATH="/opt/rh/rh-git227/root/usr/bin:${PATH}"
+
+RUN yum install -y oracle-golang-release-el7 && \
+    yum-config-manager --enable ol7_developer_golang114 && \
+    yum install -y golang-1.14.15
+
+#RUN zypper -n install gcc binutils glibc-devel-static ca-certificates git-core wget curl unzip tar vim less file xz gzip sed gawk iproute2 iptables jq
```
Comment (kminder) [inline Dockerfile.dapper:40]: Should these commented out lines be removed?

File: Dockerfile.dapper [hunk 2]
```diff
@@ -1,4 +1,7 @@
-FROM registry.suse.com/suse/sle15:15.3
+# Copyright (C) 2020, 2022, Oracle and/or its affiliates.
```
Comment (kminder) [inline Dockerfile.dapper:1]: Does this count as created in 2020?

**Review by desagar — APPROVED** [review]

File: Dockerfile.dapper (hunk 2 above)
Comment (markxnelson) [reply Dockerfile.dapper:1]: i think so, since that was in the 2.5.9 branch too

File: Dockerfile.dapper (hunk 1 above)
Comment (markxnelson) [reply Dockerfile.dapper:40]: thanks, i should remove them

**Pull request merged**